src/services/ldap/strategies/iserv-idm.js

```diff
@@ -30,8 +30,8 @@ class IservIdmLDAPStrategy extends AbstractLDAPStrategy {
 	 * (Array) roles = ['teacher', 'student', 'administrator']
 	 * @memberof GeneralLDAPStrategy
 	 */
-	verifyConfiguration(school) {
-		return this.getUsers(school);
+	verifyConfiguration() {
+		return this.getUsers();
```

Comment:
getUsers still expects the school argument

const searchString = `ou=users,${school.ldapSchoolIdentifier}`;